runtime/doc/news.txt

```diff
@@ -72,6 +72,7 @@ DEFAULTS
 
 • 'number', 'relativenumber', 'signcolumn', and 'foldcolumn' are disabled in
   |terminal| buffers. See |terminal-config| for an example of changing these defaults.
+• |linematch:40| is set by default for |diffopt|.
```

Comment:
`linematch:40` is not a valid help tag. I wonder why the lint didn't catch this :/
```suggestion
• 'diffopt' default includes "linematch:40".
```